packages/editor/src/components/blocks/select/Select.tsx

```diff
@@ -41,14 +41,20 @@ export const SelectComponent: ComponentConfig<SelectProps> = {
       subsection: 'Dynamic Options',
       label: 'Object Label',
       type: 'textBrowser',
-      options: { onlyAttributes: true, placeholder: 'Enter attribute (or leave blank to select entire object)' },
+      options: {
+        onlyAttributesOfType: data => data.dynamicItemsList,
```

Comment:
why not just true? this field should not be visible if dynamicItemslist is empty right?

Reply:
I have rewritten onlyAttributesOfType so that you can pass the type of which you want the attributes from, directly in the component, instead of just a boolean. This way I was able to reuse the onlyAttributes option from the Select-Component much easier without adjusting anything in the useAttributeBrowser. However, since I no longer need browsers for inputValue/Label at the moment, I can revert this change and work with the boolean again, if you like.

Reply:
Perfect, so I will undo this change and work (like before) with the boolean for the moment.